docs/tutorials/gcp_gpu.md

```diff
@@ -0,0 +1,195 @@
+guide to create a VM. In addition to the guidance under the Create a Linux VM
+instance heading, ensure that your VM has the following properties:
+
+*   In the Machine Configuration section, in the Machine Family options, click
+    on the **GPU** filter.
+*   In the Machine Configuration section, in the Machine family options, in the
+    GPU type option, choose **NVIDIA Tesla A100**
+    *   In the Number of GPUs option, choose **1**.
+*   In the Boot disk section, click the **Change** button.
+    *   In the Operating System option, choose **Ubuntu**.
+    *   In the Version option, choose **20.04 LTS**.
+    *   In the Size field, enter **30** (minimum).
+    *   This overrides step 3 through 5 in the Quickstart guide.
+*   In the Firewall section, ensure that both the **Allow HTTP traffic**
+    checkbox and **Allow HTTPS traffic** checkbox have marks.
+
+When Google Cloud finishes creating the VM, you can see your VM listed in the
+[Compute Instances dashboard](https://pantheon.corp.google.com/compute/instances)
+for your project.
+
+### Find out more
+
+*   [Choosing the right machine family and type](https://cloud.google.com/blog/products/compute/choose-the-right-google-compute-engine-machine-type-for-you)
+*   [Creating a VM with attached GPUs](https://cloud.google.com/compute/docs/gpus/create-vm-with-gpus#create-new-gpu-vm)
+
+## 2. Prepare your computer
+
+Use SSH to create an encrypted tunnel from your computer to your VM and redirect
+a local port to your VM over the tunnel.
+
+1.  Follow the instructions in the
+    [Installing Cloud SDK](https://cloud.google.com/sdk/docs/install)
+    documentation to install the `gcloud` command line tool.
+2.  After installation, initialize the Google Cloud environment by using the
+    `gcloud init` command. You need to provide details about your VM, such as
+    the project name and the region where your VM is located.
+    1.  You can verify your environment by using the `gcloud config list`
+        command.
+3.  Create an SSH tunnel and redirect a local port to use the tunnel by typing
+    the following command in a terminal window on your computer. Replace
+    `[YOUR_INSTANCE_NAME]` with the name of your VM.
+
+```shell
+gcloud compute ssh [YOUR_INSTANCE_NAME]
+```
+
+When the command completes successfully, your prompt changes from your local
+machine to your virtual machine.
+
+## 3. Enable your virutal machine to use the GPu
+
+1.  Install the GPU driver. In the Google Cloud documentation, in the Installing
+    GPU drivers guide, follow the steps provided:
+    *   In
+        [the Examples section](https://cloud.google.com/compute/docs/gpus/install-drivers-gpu#examples),
+        under the Ubuntu tab. Only follow the steps for Ubuntu 20.04 (steps 3a
+        through 3f).
+    *   In the
+        [Verifying the GPU driver install](https://cloud.google.com/compute/docs/gpus/install-drivers-gpu#verify-driver-install)
```

Comment:
Wording here is technically correct, but a bit difficult to parse. Consider adjusting the sentence before the bulleted list so that each item makes sense on its own.